cvat/apps/annotation/README.md

```diff
@@ -564,4 +566,60 @@ python create_pascal_tf_record.py --data_dir <path to VOCdevkit> --set train --y
       └── image2.png
   ```
 - supported shapes: Polygons
-- additional comments: the CVAT task should be created with the full label set that may be in the annotation files
\ No newline at end of file
+- additional comments: the CVAT task should be created with the full label set that may be in the annotation files
+
+### [MOT sequence](https://arxiv.org/pdf/1906.04567.pdf)
+#### Dumper
+- downloaded file: a zip archive of the following structure:
+  ```bash
+  taskname.zip/
+  ├── images/
+  |   └── img1.jpg
+  └── gt/
+      ├── labels.txt
+      └── gt.txt
+
+  # labels.txt
+  cat
+  dog
+  person
+  ...
+
+  # gt.txt
+  # frame_id, track_id, x, y, w, h, not ignored, class_id, visibility, <ignored>
```

Comment:
What "not ignored" mean? May be `# frame_id, track_id, x, y, w, h, <ignored>, class_id, visibility, <skipped>
 `

Reply:
It is `not ignored` or `included`, or `should be checked`, i.e. `1` means `include in accuracy comparison`.